tests/test_ddk.py

```diff
@@ -105,26 +142,29 @@ def test_sini_from_value(self):
             self.modelJ1713.validate()
 
     def test_sini_from_par(self):
-        temp_par_str = """PSR  J1713+0747
-               LAMBDA 256.66  1 0.001
-               BETA 30.70036  1 0.001
-               PMLAMBDA 5.2671  1  0.0021
-               PMBETA  -3.4428  1  0.0043
-               PX  0.8211  1  0.0258
-               F0  218.81  1  0.01
-               PEPOCH  55391.0
-               BINARY  DDK
-               A1 32.34  1  0.001
-               E  0.074  1  0.001
-               T0 55388.836  1  0.0002
-               PB 67.825129  1  0.0001
-               OM 176.19845  1  0.0013
-               M2  0.283395  1  0.0104
-               KOM   83.100  1  1.800
-               SINI  0.8     1  0.562
-               K96  1"""
+        test_par_str = self.temp_par_str + "\n SINI  0.8     1  0.562"
+        with pytest.raises(
+            TimingModelError,
+            match="DDK model does not accept `SINI` as input. Please use `KIN` instead.",
+        ):
+            mb.get_model(StringIO(test_par_str))
+
+    def test_stand_alone_model_params_updates(self):
+        test_par_str = self.temp_par_str + "\n KIN  71.969  1  0.562"
+        m = mb.get_model(StringIO(test_par_str))
+        verify_stand_alone_binary_parameter_updates(m)
```

Comment:
Should this test also check that the internal model has a `KIN` parameter? If not the `verify...` will still pass.

Reply:
I added two checks for the existence of `kin` before the  `verify` test